bottery/bottery.py

```diff
@@ -80,8 +80,14 @@ def configure_server(self, port):
             port=port,
         ))
 
+    def exception_handler(self, loop, context):
+        click.echo(context.get('exception'))
+        loop.default_exception_handler(context)
+        loop.stop()
```

Comment:
Should we call `self.stop()` here?

Reply:
Unfortunately, calling `self.stop()` here will cause an exception "cannot close running event loop". Also, it's semantically two different methods: `self.loop.close()` (inside `self.stop()`) tries to close event loop immediately (which is not possible in exception handler), but `loop.stop()` marks loop to be closed ASAP. I was thinking about replacing `self.loop.close()` 
 by `self.loop.stop()`, but I'm not sure if this issue a good place for such a proposal 😄

Reply:
I think `self.stop` wasn't  done the right way. There's another issue related to it (https://github.com/rougeth/bottery/issues/175). So yep, this PR isn't the best place to do discuss it indeed, but I think we should have at least a commend here saying that the stop/close should be done by another `self.stop()`.

Reply:
Ok, so I'll add a comment in this PR, and also will try to fill a PR for the #175 to stop the event loop gracefully.